client/src/components/application/Examine/RequestInfoHeader.vue

```diff
@@ -1015,7 +1014,10 @@
             $('.corp-num-spinner').removeClass('hidden')
 
             const myToken = sessionStorage.getItem('KEYCLOAK_TOKEN')
-            const url = '/api/v1/corporations/' + value
+
+            // igonre corpNum prefix 'BC' if applicable
```

Comment:
OK, thanks for the comment.

Why do we ignore BC prefix? What about other prefixes?

Reply:
In old colin, there is no 'BC' prefix for BC companies. Name-examination is an old project. Probably the validation service is still try to adapt the old corp number style.

Reply:
OK. Don't explain it to me -- explain it in the code comment so the next developer looking at this code will understand why this code is here.